### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MobiusCraftFlip**

### CODE REVIEW

**Comment by MobiusCraftFlip** [comment]
test

**Comment by MobiusMindCraft** [comment]
test123

**Review by MobiusMindCraft — APPROVED** [review]
looks g00d

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # wally-index
 test
+test123
```
Comment (MobiusMindCraft) [inline README.md]: add grammar

**Review by MobiusMindCraft — APPROVED** [review]

**Pull request merged**